cb-sol-cli/cmd/bridge.js

```diff
@@ -36,8 +36,8 @@ const registerGenericResourceCmd = new Command("register-generic-resource")
         const bridgeInstance = new ethers.Contract(args.bridge, constants.ContractABIs.Bridge.abi, args.wallet);
 
         if (args.hash) {
-            args.deposit = getFunctionBytes(args.deposit)
-            args.execute = getFunctionBytes(args.execute)
+            if (args.execute !== EMPTY_SIG) args.execute = getFunctionBytes(args.execute);
```

Comment:
If `execute` is not included but the `--deposit "requestPrice"` is set, we shouldn't try to call `getFunctionBytes(EMPTY_SIG)`, otherwise we'll unintentionally set a non empty signature for the `executeProposal` function